examples/authorization_details_jwt_vc_json.json

```diff
@@ -3,11 +3,11 @@
     "type": "openid_credential",
     "credential_configuration_id": "UniversityDegreeCredential",
     "credential_definition": {
-      "credentialSubject": {
-        "given_name": {},
-        "family_name": {},
-        "degree": {}
-      }
+      "credentialSubject": [
```

Comment:
This PR would be the opportunity to rename this to just 'claims'. The open question is whether this should then match all claims or start within "credentialSubject".

Reply:
@danielfett as I mentioned on the wgc it is possible for `claims` to be in the root of the vc in vcdm (eg.  openbadgecredential). Therefore it needs to be 
```
  "claims": {
    "path": ["credentialSubject", "given_name"], 
    "path": ["credentialSubject", "family_name"], 
    ...
   }
```

Reply:
This is a very good point and I'll make this change accordingly.

Reply:
In the latest change, I aligned the "claims" element between all credential formats, i.e., moved it one level higher, called it "claims" instead of "credentialSubject" and in the examples made the change shown in @sloops77's example.